### ISSUE TITLE
Allow to filter a specific host when calling `openio cluster unlockall`

### ISSUE THREAD

**Issue by jfsmig** [open-io/oio-sds#819]
It sounds interesting to have an option to `openio cluster unlockall` that restricts the handling to the services currently running on a given host.

Am I right, @gdelafond ?

**Comment by gdelafond** [comment]
Yes, something like : `openio cluster unlock all 172.19.12.1` will unlock all service hosted by the server 172.19.12.1.

**Comment by racciari** [comment]
I think the most coherent way to implement that functionnality would be to allow to unlock services based on the conscience-agent they are linked by. Therefore, we need to show up the conscience-agent information in the cluster informations and be able to filter on it.
Even if, for the moment, our service ids are "ip:port", we should not make IP relative actions on the cluster.

**Comment by jfsmig** [comment]
You @gdelafond and @racciari both would like filters that would fit your needs.
That's a clue we could take advantage of something expressive, as a BPF rule could be.

```
openio cluster unlock all
openio cluster unlock host 172.19.12.1
openio cluster unlock id 7e4970e1-1cde-4ede-a45c-9a688a53124a
openio cluster unlock "location dc0.room0.rack1.* or location dc0.room0.rack0.srv0"
```

Seems legit ?

**Comment by gdelafond** [comment]
Yes nice idea !
